docs/DeveloperGuide.md

```diff
@@ -245,7 +255,21 @@ The proposed theme mechanism is simply done by completely altering the base css
 If `preferences.json` is empty or the theme css file given is not valid, the default theme will be used instead and
 will replace the invalid css file specification in `preferences.json`.
 
-### \[Proposed\] Genre adding feature
+**Current Implementation**
+
+CSS files have been optimized for theme creation. Devs are only needed to edit the list of colors at the top of the CSS file
+and the source path for the background image.
+* Pros: More advanced users may be able to create their own CSS files by setting the color and changing the source of
+the background image.
+* Cons: There is a limited choice of color blocking. The bulk of the theming is made from a specified color shading.
+
+**Proposed Implementation**
```

Comment:
why is there a proposed implementation when you are describing a current feature in our app?

Reply:
@auhc99 I think this is allowed, plus the feature of allowing users to make their own themes feels like a new feature
![image](https://user-images.githubusercontent.com/69646558/140597012-5fe6da63-f31c-4b23-b57e-0923ef7d8a1e.png)

Reply:
So this all good ye? I wanted to add it as possible improvements to the project for future devs

Reply:
Then shouldn't this be under another header called [Proposed] Themes Adding Feature?

Reply:
fair.. will make the changes then